config/org.eclipse.jdt.core.prefs

```diff
@@ -100,19 +100,27 @@ org.eclipse.jdt.core.compiler.problem.unusedDeclaredThrownExceptionExemptExcepti
 org.eclipse.jdt.core.compiler.problem.unusedDeclaredThrownExceptionWhenOverriding=disabled
 #We use checkstyle rule, special cases could be suppressed by configs out of code
 org.eclipse.jdt.core.compiler.problem.unusedExceptionParameter=ignore
+#Requires us to box and unbox all conversions manually which seems unnecessary.
+org.eclipse.jdt.core.compiler.problem.autoboxing=ignore
+#Too much complaining about potential null code but tests and pitests don't show anything.
+org.eclipse.jdt.core.compiler.problem.potentialNullReference=ignore
+#Complains about any resource that leaves a method that isn't closed first.
+org.eclipse.jdt.core.compiler.problem.potentiallyUnclosedCloseable=ignore
+#Too many false positives for methods that should be static but would cause issues.
+org.eclipse.jdt.core.compiler.problem.reportMethodCanBePotentiallyStatic=ignore
+#Checkstyle doesn't mind synthetic methods.
```

Comment:
Probably we be good to show examples of violations in issue, the same is for others inspections.

Reply:
Done.